### ISSUE TITLE
[Feature] Personalized Dashboard

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#8774]
As a Moped project team member, I want to easily track and update projects I am working on so that I can keep project data current in an efficient, intuitive way. 

This will exist at the "Dashboard" link. 

**In-scope**
Basic implementation:
- My projects - links to the Project Summary for each projects I am a team member on
- Quicklinks to most-used views (right now the only option will be the "Signal projects" table)

## Implementation

**Backend**
[need to scope with devs]

**User interface**
_All adjustable based on technical constraints!_
- [x] Change "Signal projects" path to `/views/signal-projects`
- [x] At `/dashboard`, something kinda like:
![Dashboard – with checkboxes, avatars](https://user-images.githubusercontent.com/1463708/160211816-14f2a4ce-3082-49ec-b761-e3cf58c57b8d.png)

**Deets**
- Table is [Material Table](https://material-table.com/) (potentially [Core](https://material-table-core.com/), depending on #8620 
- Tried with [top tabs](https://user-images.githubusercontent.com/1463708/159979980-26f6377e-63e2-4c58-a134-6f8af88187ea.png) in the panel [like Asana](https://user-images.githubusercontent.com/1463708/160176376-e53f9235-16d0-4432-8341-ffc3a550bd6b.png) but they're built to be full-width in MUI so could get some weird behavior nesting in a panel. 
- "Status update" column should be min-width 300 and [max-width 360](https://user-images.githubusercontent.com/1463708/160001247-5f632c8f-578f-4872-ab4a-7a27274904ae.png) for optimal legibility

**Questions**

- Should status badge modal have button/menu to put on hold or delete, in addition to "Phases" table?

## Related
#8756
